modules/templates/helper.go

```diff
@@ -383,8 +383,8 @@ func NewFuncMap() []template.FuncMap {
 				if label == nil {
 					continue
 				}
-				html += fmt.Sprintf("<a href='%s/issues?labels=%d' class='ui label' style='color: %s !important; background-color: %s !important'>%s</a> ",
-					repoLink, label.ID, label.ForegroundColor(), label.Color, RenderEmoji(label.Name))
+				html += fmt.Sprintf("<a href='%s/issues?labels=%d' class='ui label' style='color: %s !important; background-color: %s !important' title='%s'>%s</a> ",
+					repoLink, label.ID, label.ForegroundColor(), label.Color, label.Description, RenderEmoji(label.Name))
```

Comment:
It will lead to XSS

Reply:
Good catch, yes this is XSS indeed. We need to escape at minimum `Description` and `Name`. There should be a suitable helper function around somehwere.

Reply:
`RenderEmoji(label.Name)` is safe, it returns the escaped `template.HTML`. I have reviewed it in another PR. 😁 

The only problem is the newly introduced `label.Description`

Reply:
Question: Wouldn't it then make sense to correct it to
```suggestion
				html += fmt.Sprintf("<a href='%s/issues?labels=%d' class='ui label' style='color: %s !important; background-color: %s !important' title='%s'>%s</a> ",
					repoLink, label.ID, label.ForegroundColor(), label.Color, RenderEmoji(label.Description), RenderEmoji(label.Name))
```
?
Then both XSS and Emojis would be handled correctly, right?

Reply:
Yup, `RenderEmoji` could do the work (but it does more, it also translates the emoji).

For `Description`, I think `html.EscapeString` is enough: https://pkg.go.dev/html#example-EscapeString , or `template.HTMLEscapeString`: https://pkg.go.dev/html/template#HTMLEscapeString

Reply:
Seems non-obvious to me that `RenderEmoji` does escaping, it would be cleaner to have it in two functions, e.g. `EscapeString(RenderEmoji(str))`, but I guess that can be a future refactor.

Reply:
I thought this would be done at the input of the label description. nice catch!